avro-converter/src/main/java/io/confluent/connect/avro/AvroData.java

```diff
@@ -818,9 +818,8 @@ private static Object maybeWrapSchemaless(Schema schema, Object value, String ty
             int precision = Integer.parseInt(precisionValue);
             baseSchema.addProp(AVRO_LOGICAL_DECIMAL_PRECISION_PROP, new IntNode(precision));
           } else {
-            baseSchema
-                .addProp(AVRO_LOGICAL_DECIMAL_PRECISION_PROP,
-                         new IntNode(CONNECT_AVRO_DECIMAL_PRECISION_DEFAULT));
+            baseSchema.addProp(AVRO_LOGICAL_DECIMAL_PRECISION_PROP,
```

Comment:
seems like we are introducing an artificial change. This code remained unchanged praticaly